CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/plugins/DataAccessViewWelcomePlugin.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.welcome.plugins;
+
+import au.gov.asd.tac.constellation.plugins.PluginInfo;
+import au.gov.asd.tac.constellation.plugins.PluginType;
+import au.gov.asd.tac.constellation.views.dataaccess.panes.DataAccessViewTopComponent;
+import au.gov.asd.tac.constellation.views.welcome.WelcomePluginInterface;
+import au.gov.asd.tac.constellation.views.welcome.WelcomeTopComponent;
+import javafx.geometry.Pos;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.VBox;
+import javax.swing.SwingUtilities;
+import org.openide.util.NbBundle;
+import org.openide.windows.TopComponent;
+import org.openide.windows.WindowManager;
+
+/**
+ * The Open Data Access View for the Welcome Page.
+ *
+ * @author Atlas139mkm
+ */
+@PluginInfo(pluginType = PluginType.CREATE, tags = {"WELCOME"})
```

Comment:
I don't think CREATE is the right plugin type here. The other welcome plugins that open views don't seem to have types yet so you could potentially just omit. NAVIGATION potentially looks like the best option if you want to keep it though

Reply:
yep sounds good, apologies I was just mimicking one of the other files.